hAMRonization/Interfaces.py

```diff
@@ -199,6 +199,13 @@ def generate_tool_subparser(subparser, analysis_tool):
         tool_parser.add_argument(f"--{field}", required=True,
                                  help=f"Input string containing the {field} "
                                  f"for {analysis_tool}")
+    # any missing optional fields need supplied as CLI argument
+    optional_metadata = \
```

Comment:
Its a good solution for a clunk situation but does add complexity to an already kinda complex CLI generation